README.md

```diff
@@ -44,6 +44,17 @@ Insert `vwifi` driver:
 sudo insmod vwifi.ko ssid_list='[MyHomeWiFi][MyWifi_1][MyWifi_2]'
 ```
 
+Check wiphy:
+```shell
+iw list
+```
+
+Create two network interfaces (station mode) on top of wiphy "owl"
+```shell
+sudo iw phy owl interface add owl0 type managed
+sudo iw phy owl interface add owl0sink type managed
```

Comment:
Do we still need the `owl0sink` interface? If not, let us remove `owl0sink`. And the other document or scripts.

Reply:
Ok. I'll remove it.